gemfiles/rails_6.0/Gemfile

```
@@ -1,5 +1,5 @@
 source "https://rubygems.org"
 
-gem "rails", "~> 6.0.2"
+gem "rails", "~> 6.0"
```

this will include 6.1, should be changed back to patch level

```suggestion
gem "rails", "~> 6.0.0"
```